### ISSUE TITLE
display cropping issue in memory trace visualization

### ISSUE THREAD

**Issue by fstonedahl** [StanWarford/pep9suite#118]
On Linux (HiDPI screen) - the stack memory visualization (based on traced tags) is cutting off the first digit of the memory addresses -- as shown here (stepping through Figure 6.4):

![image](https://user-images.githubusercontent.com/770633/160756827-6b9c58bb-49c9-42a1-aa58-67623453a1a5.png)

(Changing the "Font Size" does not seem to affect the size of the memory visualization.)

**Comment by Matthew-McRaven** [comment]
Good catch! I suspect that I didn't anticipate users switching to larger-than-default fonts here.

This is being tracked, and will be fixed in the next version.

**Comment by fstonedahl** [comment]
I've confirmed that this is a "HiDPI screen issue".   (It might also just be a Linux-only HiDPI issue - I don't have a HiDPI windows machine to test it on right now.)

More information: 

In Mint Linux, when I change my desktop user interface scaling to Normal 1:1 instead of Double 2:1, then the memory visualization text doesn't get cut off... it's just really small on my hi-resolution laptop screen).

And, when I am in my normal double scaling setting, it looks like it fixes the issue for me if I set this QT environment variable before running Pep9:

`QT_SCREEN_SCALE_FACTORS=2 pep9`

If there's some way to fix it on your end so that it scales correctly "out of the box", but at least I have a workaround now in any case.

**Comment by fstonedahl** [comment]
Update: there might also be a problem with printing the source code, where the print font is too large and getting clipped off.  

(example of clipping problem from a print-to-pdf)
![image](https://user-images.githubusercontent.com/770633/162369110-db8f7419-a04e-44ef-a2f9-bc24bc7cad2c.png)

Again, I suspect "HiDPI" issues are at play.  (However, this printing issue occurred even after I had set the QT_SCREEN_SCALE_FACTORS=2 environment variable... so I guess that wasn't a panacea.)

**Comment by fstonedahl** [comment]
At one point I changed the font size, and it did work without clipping... so I'm not quite sure what the printing-related issue is. Not a big issue in any case...
